src/content/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips.mdx

```diff
@@ -25,19 +27,15 @@ Because of this, ensure your firewall allows their network requests through it.
 
 ## IP addresses are not personal data [#not-personal-data]
 
-Minions are deployed on servers, and the agents are expected to be activated using non-personal credentials. IP addresses associated with minions running on servers are not personal data under data protection and privacy laws. For more information, see the [Synthetic's security documentation](/docs/synthetics/new-relic-synthetics/getting-started/security-new-relic-synthetics).
+Minions are deployed on servers, and the agents are activated using non-personal credentials. IP addresses associated with minions running on servers are not personal data under data protection and privacy laws. For more information, see the [Synthetic's security documentation](/docs/synthetics/new-relic-synthetics/getting-started/security-new-relic-synthetics).
 
 ## Daily JSON listings for IP addresses [#dynamic-ip-addresses]
 
 IP addresses for released locations are subject to change. If a change is needed, we'll attempt to proactively notify customers prior to any changes via e-mail. You can also check the [Explorers Hub](https://discuss.newrelic.com/) for updates.
 
 <Callout variant="important">
-   Synthetics is [adding new IP addresses on July 15 2021](/docs/synthetics/synthetic-monitoring/administration/new-synthetic-public-minion-ips).
-</Callout>
-
-<Callout variant="tip">
-  In the S3 URL paths for this feature, `production` represents US-based accounts and `eu` represents [EU-based accounts](/docs/using-new-relic/welcome-new-relic/get-started/our-eu-us-region-data-centers).
-</Callout>
+    By the end of March 2022, [new IP address ranges](/docs/synthetics/synthetic-monitoring/administration/new-synthetic-public-minion-ips) may require that you reconfigure your firewall settings.
```

Comment:
I'd get as granular as you can here. This is fine for now, but I'd update it to the exact date as soon as it's known.

Reply:
We don't want to remove this callout about production vs eu in the JSON listings. While the new IP address ranges will include both our US and EU locations, the existing IPs will continue to be used for months too and this production / eu breakdown is important for those IPs.